modules/monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-apis.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_{context}"]
+= Querying metrics by using the federation endpoint for Prometheus
+
+You can use the federation endpoint to query a cluster's scraped metrics from a network location outside the cluster. 
+To do so, access the Prometheus `/federate` endpoint for the cluster via an {product-title} route. 
+
+You can access the `/federate` endpoints for default platform monitoring and for user-defined projects.
```

Comment:
better change endpoints to routes

Reply:
Maybe combine with the previous paragraph? e.g.

```
You can use the federation endpoint to scrape platform and user-defined metrics from a network location outside the cluster.
```

Reply:
Revised per @simonpasquier's suggestion